gcp/appengine/frontend_handlers.py

```diff
@@ -190,6 +191,40 @@ def add_source_info(bug, response):
   response['source_link'] = response['source']
 
 
+def add_related_aliases(bug: osv.Bug, response):
+  """Add links to other osv entries that's related through aliases"""
+  # Add links to other entries if they exist
+  aliases = {}
+  for alias in bug.aliases:
+    if len(
+        bug.aliases) <= 8:  # only if there aren't too many, otherwise skip this
```

Comment:
nit: can you put 8 into a global CONSTANT above?

Reply:
Also the line break is a little strange. If this is because of the comment, we could put the comment before line 201 instead.